spacepy/toolbox/__init__.py

```diff
@@ -1467,6 +1471,7 @@ def indsFromXrange(inxrange):
     if not isinstance(inxrange, xrange): return None
     valstr = inxrange.__str__()
     if ',' not in valstr:
+        # TODO, can't figure out in what case this code is run, every case is False then off to else.
```

Comment:
Probably means we're lacking a test, then. `spacepy.toolbox.indsFromXrange(xrange(5))`

Reply:
I can;t wait to drop xrange entirely... #26

Reply:
But the new `range` is just old `xrange` so this function will stay around.